command/git.go

```diff
@@ -264,9 +270,27 @@ func (c *gitFetchProject) buildCloneCommand(conf *model.TaskConfig, opts cloneOp
 		}
 	}
 
+	gitCommands = append(gitCommands, c.getConfigCommands()...)
```

Comment:
Is this something we only want to do if isMBP or does it not matter?

Reply:
I considered just setting it in the case where we need it (when we run git am) but it seems like a good idea to set it in every case since git sometimes refuses to do random things when it's not set.